profiles/video.jsonld

```diff
@@ -0,0 +1,551 @@
+        "any": [
+          "https://w3id.org/xapi/video/verbs/played",
+          "https://w3id.org/xapi/video/verbs/paused",
+          "https://w3id.org/xapi/video/verbs/seeked"
+        ]
+        }]
+      }
+  ],
+  "patterns": [
+      {
+         "id": "_:generalpattern",
+         "primary": true,
+          "name": {
+              "en": "General Pattern"
+          },
+          "definition": {
+              "en": "This pattern describes the overall sequence of statements sent over the course of an attempt"
+          },
+          "sequence": ["https://w3id.org/xapi/video/templates#initialized", "_:optionalmiddlestatements", "https://w3id.org/xapi/video/templates#terminated"]
+      },
+      {
+          "@id": "_:played-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played"]
+      },
+      {
+          "@id": "_:played-completed-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#completed"]
+      },
+      {
+          "@id": "_:played-paused-completed-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#paused", "https://w3id.org/xapi/video/templates#completed"]
+      },
+      {
+          "@id": "_:played-paused-seeked-completed-pattern",
+          "alternates": ["_:generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#paused", "https://w3id.org/xapi/video/templates#seeked", "https://w3id.org/xapi/video/templates#completed"]
+      },
+      {
+          "@id": "_:played-paused-seeked-volumechange-completed-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#paused", "https://w3id.org/xapi/video/templates#seeked", "https://w3id.org/xapi/video/templates#volumechange", "https://w3id.org/xapi/video/templates#completed"]
+      },
+      {
+          "@id": "_:all-activities-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played", "https://w3id.org/xapi/video/templates#paused", "https://w3id.org/xapi/video/templates#seeked", "https://w3id.org/xapi/video/templates#volumechange", "https://w3id.org/xapi/video/templates#screenchange", "https://w3id.org/xapi/video/templates#completed"]
+      },
+      {
+          "id": "_:optionalmiddlestatements",
+          "type": "Pattern",
+			    "zeroOrMore": ["_:played-pattern", "_:played-completed-pattern", "_:played-paused-completed-pattern", "_:played-paused-seeked-completed-pattern", "_:played-paused-seeked-volumechange-completed-pattern", "_:all-activities-pattern"],
```

Comment:
can you have multiple "zeroOrMore" properties, like this? 

`      {
          "id": "https://w3id.org/xapi/video/patterns#optionalmiddlestatements",
          "type": "Pattern",
			    "zeroOrMore": ["https://w3id.org/xapi/video/patterns#played-pattern"], 
          "zeroOrMore": ["https://w3id.org/xapi/video/patterns#played-completed-pattern"], 
          "zeroOrMore": ["https://w3id.org/xapi/video/patterns#played-paused-completed-pattern"], 
          "zeroOrMore": ["https://w3id.org/xapi/video/patterns#played-paused-seeked-completed-pattern"], 
          "zeroOrMore": ["https://w3id.org/xapi/video/patterns#played-paused-seeked-volumechange-completed-pattern"], 
          "zeroOrMore": ["https://w3id.org/xapi/video/patterns#all-activities-pattern"]

      }`